addons/api/tests/dummy/mirage/config.js

```diff
@@ -1,22 +1,5 @@
-export default function () {
-  // These comments are here to help you get started. Feel free to delete them.
-  /*
-    Config (with defaults).
+import { createServer } from 'miragejs';
 
-    Note: these only affect routes defined *after* them!
-  */
-  // this.urlPrefix = '';    // make this `http://localhost:8080`, for example, if your API is on a different server
-  // this.namespace = '';    // make this `/api`, for example, if your API is namespaced
-  // this.timing = 400;      // delay for each request, automatically set to 0 during testing
-  /*
-    Shorthand cheatsheet:
-
-    this.get('/posts');
-    this.post('/posts');
-    this.get('/posts/:id');
-    this.put('/posts/:id'); // or this.patch
-    this.del('/posts/:id');
-
-    https://www.ember-cli-mirage.com/docs/route-handlers/shorthands
-  */
+export default function (config) {
+  return createServer({ ...config });
```

Comment:
This was to fix the deprecation message thrown by mirage when running tests

Reply:
Nice! yeah I think here we do not need to use `models` neither `serializers` KISS!